lib/config/options/index.ts

```diff
@@ -2615,6 +2615,15 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: false,
   },
+  {
+    name: 'checkedBranches',
```

Comment:
shouldn't this be a global only config option? it doesn't make sense to have in presets or repo config file

Reply:
i think so 
ill add globalOnly: true